DependencyInjection/Configuration.php

```diff
@@ -783,6 +790,17 @@ public function getConfigTreeBuilder()
                             ->scalarNode('formatter')->end()
                             ->booleanNode('nested')->defaultFalse()->end()
                         ->end()
+                        ->beforeNormalization()
+                            ->always(static function ($v) {
+                                if (empty($v['console_formatter_options']) && !empty($v['console_formater_options'])) {
+                                    $v['console_formatter_options'] = $v['console_formater_options'];
+                                }
+
+                                unset($v['console_formater_options']);
```

Comment:
if you unset the node here, the deprecation won't ever be triggered by the node itself

Reply:
Fixed